library/aes.c

```diff
@@ -1303,8 +1331,10 @@ int mbedtls_aes_crypt_cfb8( mbedtls_aes_context *ctx,
 
         memcpy( iv, ov + 1, 16 );
     }
-
-    return( 0 );
+end:
+    if( ret != 0 )
+        mbedtls_platform_zeroize( out, orig_length );
```

Comment:
Do we really need to zeroize `out`? Nothing has been written to `out` yet.

Reply:
Same answer as https://github.com/ARMmbed/mbedtls/pull/1096/files#r222589243

Reply:
OK